### ISSUE TITLE
After the file upload, click on the file list, there is still a bug, there is no record of file upload in the database. Why is this?

### ISSUE THREAD

**Issue by xiaodupi-zyq** [lucky-luk3/Grafiki#2]
After the file upload, click on the file list, there is still a bug, there is no record of file upload in the database. Why is this?

**Comment by lucky-luk3** [comment]
You are right, the problem was solved in the dev branch. If you can test it, it would be nice. Thank you.

**Manual solution**
Create table "app_file"
```
CREATE TABLE  "app_file"
(
    id SERIAL,
    name VARCHAR(255),
    evtx VARCHAR(255),
    processed VARCHAR(255),
    test VARCHAR(255)
);
```
Change /app/templates/base.html line 50 to:
```
<a class="nav-link" href="{% url 'upload_file' %}">File Upload</a>
```

Change app/views.py line 78 to:
```
return redirect(file_list)
```

Please, let me know if the problem was solved.

**Comment by xiaodupi-zyq** [comment]
Thank you! I will try to run dev

**Comment by xiaodupi-zyq** [comment]
Can I use the data from the APT29 simulation plan in the mordor dataset to generate a graph？https://github.com/hunters-forge/mordor

**Comment by lucky-luk3** [comment]
No, I'm sorry, Mordor has different versions of filebeat and in APT29 dataset, other tool was used. 
I have to change some things to process this dataset.
Mordor datasets that grafiki can process are included in examples. :)

**Comment by xiaodupi-zyq** [comment]
Is it possible to generate a graph after extracting and cleaning the data in the mordor data set?

**Comment by lucky-luk3** [comment]
I think that you can delete lines without problem if this dataset has the appropriate format.
Also you can import mordor dataset in helk and use grafiki to request information to helk's elastic.

It's important to keep in mind that not all of mordor dataset can be used now in grafiki.

**Comment by xiaodupi-zyq** [comment]
Thank you, I will try and give you feedback
